### PR TITLE
new PR for ya

### PR DESCRIPTION
**Pull request by thornd3**

### CODE REVIEW

**Comment by jc-yap** [comment]
I don't know how to do inline comments so I'll just write an essay here.

It was a little hard to follow what you were doing with the code. You could add some commentary between the code to explain what you're doing. It might look better to write the open-ended answers as text (you can change between code and text in the upper bar of Matlab livescript) instead of in comments. 

I'm also not sure if you did a hypothetical question for the third part and answered the last 2 questions of the lab.

I'm not sure if it's necessary but you could add a couple more plots of the rayleigh distribution with different parameters. Also it looks like you wanted to do that with the poisson distribution but one plot overrode the other, so I can only see one plot there. Changing the code for the first plot to:
subplot(1,2,1) fixes this.

There are a couple of times earlier on when you use subplot when I think you could just use the plot command. Subplot makes them smaller and a little harder to see, but it's a good idea for the poisson one where you want to compare the two plots.

**Comment by thornd3** [comment]
@oliviatessa i also requested a review, let me know if that's not necessary

File: PHYS_434_Lab1.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
...N8FAQAAYMsqr9QU5dC+XQLwd0EAAADY1O7fv698Pq+xsTFNTk7qoXKlpnKlpii5bFIA/i4IAAAAm9by8rKOHz+uy5cva3Z2ViMjIzp58qTq9brKKw8UJZ1KCMA/CgIAAMCmdfHiRe3cuVPj4+O6dOmS8vm8FhYWNDo6qqmFqqKkkwltde+++66AJxEEAACATaler2tiYkKnTp1SPB5X0549e3Ts2DHduHFDhdKKouSySW117733noAnEQQAAIBNaW5uTvV6XXv37tVqBw4c0Pz8vN4+nNa5Ixnlsrv0KP/3+f9TZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMZiYzk5nJzGRmMjOZmcxMTWYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZycxkZjIzmZnMTGYmM5OZ6dVXXxWeThAAAAA2pWq1qqauri6ttmPHDjX9b19M6OzRjD4aPqj/+X/9H1r8z3/TR8MHdO5IRrnsLn0y/f/K3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u9xd7i53l7vL3eXucne5u95//33h6QQBAABgU2o0GnoS6VRCuWxSZ49m9NHwQQH4Z0EAAADYlGKxmJpqtZpWazQaaorFYgLwZIIAAACwKWUyGTXNz89rteXlZW3fvl3xeFwAnkwQAAAANqV0Oq3du3erWCxqtenpaXV3dwvAkwsCAADApnXixAldvXpVd+7cUVM+n9ft27fV398vAE8uCAAAAJvW8PCwDh8+rL6+PnV2dur06dM6c+aMent7BeDJB
```
Comment (oliviatessa) [inline PHYS_434_Lab1.html:76]: When only plotting one thing, use plot(x,y). Makes it easier to read.

Comment (oliviatessa) [inline PHYS_434_Lab1.html:82]: Explain what you are doing for the following 18 lines of code. What are the inputs of erfc(), normcdf(), and norminv()? Why are you using them and what do your results mean?

Comment (oliviatessa) [inline PHYS_434_Lab1.html:99]: That's when it appears, but why does it appear? What is its significance?

Comment (oliviatessa) [inline PHYS_434_Lab1.html:130]: Does this make sense? Why would you use these integration bounds?

Comment (oliviatessa) [inline PHYS_434_Lab1.html:136]: Make sure you discuss the results of each calculation. Do they make sense to you? Why?

Comment (oliviatessa) [inline PHYS_434_Lab1.html:160]: Good in depth discussion! You need a lot more of this throughout your write-up.

**Comment by oliviatessa** [comment]
Most important note going forward: you need to go a lot deeper in your explanations. Some parts of your lab weren't explained at all.
